packages/table/src/react/__stories__/react-table.story.tsx

```diff
@@ -747,19 +747,11 @@ const selectionHook = (hooks: Hooks<any>) => {
 
 const SelectionCell: React.FC<CellProps<any>> = props => {
   const { row } = props
-  return <TableCheckbox {...row.getToggleRowSelectedProps()} />
+  return <Checkbox {...row.getToggleRowSelectedProps()} />
 }
 
 const SelectionHeader: React.FC<HeaderProps<any>> = props => {
   const { getToggleAllRowsSelectedProps } = props
   const style = { width: 1 }
-  return <TableCheckbox {...getToggleAllRowsSelectedProps({ style })} />
-}
-
-interface TableCheckboxProps extends Omit<HTMLPropsFor<'input'>, 'ref'> {
-  indeterminate?: boolean
-}
-const TableCheckbox: React.FC<TableCheckboxProps> = props => {
```

Comment:
Confirmed, maybe? This story doesn't seem clickable. http://localhost:6006/?path=/story/components-table--row-selection But didn't cross-check with master.

Reply:
Can put it back. It's not part of the Table component. It's more something that should be in docs. Also was throwing type errors because of that and it wasn't typed properly to begin with.  Strict was not set to true the HTMLPropsFor refactor exposed underlying type issues as a result which is why I set it too true in all packages.